packages/testing/src/api/index.ts

```diff
@@ -64,25 +64,32 @@ export function initializeTestApp(options: any): admin.app.App {
   );
 }
 
-export function loadDatabaseRules(options: any): void {
-  if (!('databaseName' in options)) {
+export type LoadDatabaseRulesOptions = {
+  databaseName: String,
+  rules: String,
```

Comment:
can you add a test for only specifying rules? also perhaps one for utf8 encoding as well.

Reply:
We don't have a good way of testing that these requests succeed, only that they fail. If I pass in valid rules or a valid rulesPath, the method still throws an exception because there's no emulator listening.